### PR TITLE
Add sigmoid activation function and change loss

### PR DESCRIPTION
**Pull request by Jaime-prog**
Defines function of training 
Modifies the optimizer in order to be binary cross-entropy

### CODE REVIEW

**Review by Javier1257 — APPROVED** [review]
It looks good to me
